src/app/components/drawer/SavedSearches/SavedSearchActions.js

```diff
@@ -165,24 +179,24 @@ const ProjectActions = ({
         open={Boolean(anchorEl)}
         onClose={handleClose}
       >
-        <MenuItem className="project-actions__edit" onClick={() => { setShowEditDialog(true); }}>
+        <MenuItem className="saved-search-actions__edit" onClick={() => { setShowEditDialog(true); }}>
           <ListItemText
             primary={
               <FormattedMessage
                 defaultMessage="Rename"
                 description="'Rename' here is an infinitive verb"
-                id="projectActions.rename"
+                id="savedSearchActions.rename"
```

Comment:
Please let's keep existring string ids as they were to avoid unnecessary re-translation work and costs.

Reply:
Done